data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -2365,7 +2365,7 @@
 },{
   "type" : "recipe",
   "result": "slime_scrap",
-  "byproducts": "slime_scrap",
+  "byproducts": [ [ "slime_scrap" ] ],
```

Comment:
This one could probably be changed into a multiplier.
We could also have a warning on byproduct being the same as result.